### ISSUE TITLE
Suppress Warnings from pint.util

### ISSUE THREAD

**Issue by davemfish** [natcap/invest#631]
We have all these warnings logged when we import `spec_utils.py`

```
08/31/2021 08:36:43  pint.util          WARNING  Redefining 'pixel' ()
08/31/2021 08:36:43  pint.util          WARNING  Redefining 'survey_foot' ()
08/31/2021 08:36:43  pint.util          WARNING  Redefining 'sft' ()
08/31/2021 08:36:43  pint.util          WARNING  Redefining 'henry' ()
08/31/2021 08:36:43  pint.util          WARNING  Redefining 'hour' ()
08/31/2021 08:36:43  pint.util          WARNING  Redefining 'h' ()
08/31/2021 08:36:43  pint.util          WARNING  Redefining 'hr' ()
08/31/2021 08:36:43  pint.util          WARNING  Redefining 'year' ()
08/31/2021 08:36:43  pint.util          WARNING  Redefining 'yr' ()
08/31/2021 08:36:43  pint.util          WARNING  Redefining 'a' ()
08/31/2021 08:36:43  pint.util          WARNING  Redefining 'julian_year' ()
```

Since we decided it's a good idea to redefine those units, I think we should suppress the warnings.

**Comment by emlys** [comment]
I believe the warnings would also go away if we used a custom unit definitions file instead of programmatically redefining

**Comment by davemfish** [comment]
Fixed in #643
